static/app/utils/theme/theme.chonk.tsx

```diff
@@ -0,0 +1,185 @@
+const _chonkShared = {
+  space: {
+    // @TODO(jonasbadalic): none doesn't need to exist
+    // none: 0,
+    subMicroscoptic: 1,
+    microscoptic: 2,
+    mini: 4,
+    small: 6,
+    medium: 8,
+    large: 12,
+    huge: 16,
+  },
+  borderRadius: {
+    // @TODO(jonasbadalic): none doesn't need to exist
+    // none: 0,
+    subMicroscoptic: 1,
+    microscoptic: 2,
+    mini: 3,
+    small: 4,
+    medium: 5,
+    large: 6,
+  },
+};
+
+const _chonkLightColors = {
+  // @TODO(jonasbadalic): add explanation about static and dynamic color differences and intended usage
```

Comment:
Basic explanation from my side.

The static colour scales are used for certain components or specific style variants that don't change according to the theme. 

For example; 
- Component: <Avatar/>
- Style Variant: <Button priority="primary" />